### PR TITLE
fix(docs): update contribution guide

### PR DESCRIPTION
**Pull request by kobenguyent**
I'm following this guide to run local tests however, I'm struggling to make it up and running.
https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md

This PR aims to update docs to help anyone who gets into trouble like me.

### CODE REVIEW

**Comment by PeterNgTr** [comment]
@microsoft-github-policy-service agree

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 ### Getting Code
 
+Make sure you're running node >= 14.x and npm >= 8, if not upgrading your npm
+
+```bash
+npm i -g npm@8
```
Comment (mxschmitt) [inline CONTRIBUTING.md:23]:
```suggestion
npm i -g npm@latest
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,7 +33,8 @@ cd playwright
 2. Install dependencies
 
 ```bash
-npm install
+npm ci
+npm ci @playwright/test
```
Comment (mxschmitt) [inline CONTRIBUTING.md:37]:
```suggestion
```

That's not required.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -39,7 +46,7 @@ npm run build
 4. Run all Playwright tests locally. For more information about tests, read [Running & Writing Tests](#running--writing-tests).
 
 ```bash
-npm test
+npm run local-test
```
Comment (mxschmitt) [inline CONTRIBUTING.md:49]:
```suggestion
npm test
```

Why does npm test not work for you?

File: package.json [hunk 4]
```diff
@@ -27,6 +27,7 @@
     "ttest": "node ./tests/playwright-test/stable-test-runner/node_modules/@playwright/test/cli test --config=tests/playwright-test/playwright.config.ts",
     "ct": "playwright test tests/components/test-all.spec.js --reporter=list",
     "test": "playwright test --config=tests/library/playwright.config.ts",
+    "local-test": "node packages/playwright-core/lib/cli/cli.js test --config=tests/library/playwright.config.ts",
```
Comment (mxschmitt) [inline package.json:30]:
```suggestion
```

That's the same as just "test" npm will figure out the paths so playwright resolves to the playwright package in our monorepo.

**Comment by PeterNgTr** [comment]
thanks @mxschmitt!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -17,6 +17,14 @@
 
 ### Getting Code
 
+Make sure you're running Node.js 14+ and NPM 8+, to verify do:
```
Comment (mxschmitt) [inline CONTRIBUTING.md:20]:
```suggestion
Make sure you're running Node.js 14+ and NPM 8+, to verify and upgrade NPM do:
```

**Pull request merged**